tests/e2e/seed/scripts/deploy.ts

```diff
@@ -26,6 +26,7 @@ import { MinterSetPriceERC20V5__factory } from "../contracts/factories/MinterSet
 import { MinterSetPriceMerkleV5__factory } from "../contracts/factories/MinterSetPriceMerkleV5__factory";
 import { MinterSetPriceHolderV5__factory } from "../contracts/factories/MinterSetPriceHolderV5__factory";
 import { MinterSEAV1__factory } from "../contracts/factories/MinterSEAV1__factory";
+import { MinterRAMV0__factory } from "../contracts/factories/MinterRAMV0__factory";
```

Comment:
you might want to pull the latest MinterRAMV0.json, because there have been some bug fixes in PR #1383